platform/pulp/app/settings.py

```diff
@@ -103,7 +103,7 @@
     },
 ]
 
-WSGI_APPLICATION = 'wsgi.application'
+WSGI_APPLICATION = 'pulp.app.wsgi.application'
```

Comment:
@ammaritiz the pulp package was refactored to be pulpcore, so this needs to be pulpcore.app.wsgi.application